src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -353,6 +421,19 @@ private IEnumerable<FileInfo> FindTemplateFilesInDirectory(DirectoryInfo directo
             return armTemplates.Concat(bicepTemplates);
         }
 
+        private IEnumerable<FileInfo> FindParameterFileForTemplate(FileInfo template)
+        {
+            var parametersFiles = template.Directory.GetFiles(
+                Path.GetFileNameWithoutExtension(template.Name) + ".parameters*.json",
```

Comment:
```suggestion
                Path.GetFileNameWithoutExtension(template.Name ".parameters") + "*.json",
```
Not sure if this is better or the way you wrote it is better

Reply:
Tried to use it your way but that doesn't build, so for now I'll keep it the way it is 😉